### PR TITLE
Rooter post front

### PR DESCRIPTION
**Pull request by ArchStanton987**
This PR features the new PostModal component (front-end only). A modal to post a new message appears when we click on "poster un message".

### CODE REVIEW

File: studips/src/component/PostModal.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '../App.css';
+
+function PostModal({ isPostModalVisible }) {
+	return isPostModalVisible ? (
```
Comment (rocambille) [inline studips/src/component/PostModal.jsx]: use a more suited operator: if your "else" is null, prefer `&&` ;)

File: studips/src/component/PostModal.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		<div className='overlay'>
```
Comment (rocambille) [inline studips/src/component/PostModal.jsx]: change tabs for spaces, and recheck indentation

**Review by rocambille — CHANGES_REQUESTED** [review]

File: studips/src/component/PostModal.jsx (hunk 1 above)
Comment (ArchStanton987) [reply studips/src/component/PostModal.jsx]: Fixed :)

File: studips/src/component/PostModal.jsx (hunk 2 above)
Comment (ArchStanton987) [reply studips/src/component/PostModal.jsx]: Fixed as well.

**Pull request merged**